benchmark/esy.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "brisk-reconciler-bench",
+  "version": "0.0.1",
+  "license": "MIT",
+  "esy": {
+    "build": [
+      "dune build -p brisk-reconciler-bench"
+    ]
+  },
+  "scripts": {
+    "fmt": "dune build @fmt --auto-promote",
+    "clean": "dune clean",
+    "run": "esy x benchmark.exe"
+  },
+  "dependencies": {
+    "@esy-ocaml/reason": ">=3.4.0",
+    "@opam/dune": "^1.6.2",
+    "ocaml": "~4.7.1000",
+    "brisk-reconciler": "*"
+  },
+  "devDependencies": {
+    "@opam/merlin": "^3.2.2",
+    "@opam/core_bench": "*",
```

Comment:
Unfortunately `core_bench` doesn't build on Windows ☹️ Might be problematic for other platforms - just a heads up!

Reply:
oh snap! Fortunately we don't have to include it in CI or main package so we should be good to go.